test/spec/props.test.js

```diff
@@ -1145,8 +1145,8 @@ describe('followCursor', () => {
   // should be within that
   const first = { clientX: 317, clientY: 119 }
   const second = { clientX: 240, clientY: 500 }
-  const fakeSize = 100
-  const halfFakeSize = fakeSize / 2
+  const size = 0
+  const halfSize = size / 2
```

Comment:
0 / 2? It is for documentation?

Reply:
`.offsetWidth/Height` are 0 in Jest since it doesn't render anything... just copying the source code